### PR TITLE
feat(msteams): Add default installation_type for new integrations

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective:
New rows in sentry_integration created for MS Teams should have installation_type field in the metadata column. It should default to “team“ right now.

### CODE REVIEW

File: src/sentry/integrations/msteams/integration.py [hunk 1]
```diff
@@ -94,6 +94,8 @@ def build_integration(self, state):
                 "access_token": token_data["access_token"],
                 "expires_at": token_data["expires_at"],
                 "service_url": service_url,
+                # TODO: Default to 'team' installation type for now.
```
Comment (ceorourke) [inline src/sentry/integrations/msteams/integration.py:97]: I assume you're going to update this right after the migration but isn't the todo to make it `installation_type` (or `metadata["installation_type"]` idk)?

Comment (ceorourke) [reply src/sentry/integrations/msteams/integration.py:97]: In the future the installation type will be either team or user, right? So right now it's hard coded as team but at some point it'll be a variable, and the todo just says what it's doing right now, not what it needs to be changed to be. It's not a huge deal if you are the one who's going to change it in the very near future, but if someone else looked at this it's unclear what needs to change.

Comment (ceorourke) [reply src/sentry/integrations/msteams/integration.py:97]: So maybe put "TODO: Read installation_type from metadata and set as team or user" is more clear? I might be misunderstanding though

**Review by ceorourke — APPROVED** [review]
lol sorry for being nitpicky on the todo comment, was just making sure I understood the change

**Review by 97varun — APPROVED** [review]

**Pull request merged**